docs/getting-started/summaryofchanges.md

```diff
@@ -46,6 +46,11 @@ The following features and enhancements were added.
 #### Zowe installation
 #### API Mediation Layer
 #### Zowe App Server
+
+- Using a cross-memory server without `REUSASID=YES` may result in an ASID shortage. This pull-request adds a check that will print a warning if `REUSASID=YES` is not detected [#145](https://github.com/zowe/zowe-common-c/pull/145)
+- The `InstanceID` parameter within the server predates the environment variable. Therefore, in order to not disturb pre-existing parameters, the server will only use the environment variable if it is non-default. [#130](https://github.com/zowe/zlux-app-server/pull/130)
+- Package size of the editor has been significantly reduced by removing assets that are unlikely to be used, specifically, `.map ` files  [#160](https://app.zenhub.com/workspaces/zowe-apps-5ce5829c1c7e0448d98d961e/issues/zowe/zlux-editor/160)
```

Comment:
That's an opinion, it would be more correct to say:

The packaged size of the Editor has been significantly reduced by removing uncompressed versions of files that have compressed variants, and `.map` files which were used for development debugging.